src/frameworks/angular/utils.ts

```diff
@@ -623,3 +571,13 @@ function throwCannotDetermineTarget(error?: Error): never {
     { original: error },
   );
 }
+
+/**
+ * Extracts the builder type from a full builder string (everything after the colon)
+ * @example
+ * getBuilderType("@angular-devkit/build-angular:browser") // returns "browser"
+ */
+function getBuilderType(builder: string): string {
+  const colonIndex = builder.lastIndexOf(":");
+  return colonIndex >= 0 ? builder.slice(colonIndex + 1) : builder;
+}
```

Comment:
should we return null when it's not valid instead of returning a possibly invalid builder type?

Reply:
done 🫡